src/observable.js

```diff
@@ -64,6 +64,9 @@ export class Observable {
    * @param {TYPE} event
    */
   fire(event) {
+    if (this.handlers_.length == 0) {
```

Comment:
Paranoia. I think that it does avoid some binding costs in the lines below. Some of the event handlers could be pretty chatty. But I can remove if it's overly paranoid.